includes/wf_crm_webform_preprocess.inc

```diff
@@ -183,12 +183,21 @@ class wf_crm_webform_preprocess extends wf_crm_webform_base {
         'scope' => 'footer',
       );
       $page_id = $this->data['contribution'][1]['contribution'][1]['contribution_page_id'];
-      $contributionCallbackQuery = array('reset' => 1, 'id' => $page_id, 'qfKey' => $this->getQfKey(), 'snippet' => 4);
+      if ($this->civicrm_version >= '4.7') {
+        $currency = '$'; // FIXME
```

Comment:
Hmm, that's probably the site-wide default rather than the currency of this contribution page. The webform should follow whatever currency the page is set to.